trie/zk_trie_test.go

```diff
@@ -148,3 +153,80 @@ func TestZkTrieConcurrency(t *testing.T) {
 	// Wait for all threads to finish
 	pend.Wait()
 }
+
+func tempDBZK() (string, *Database) {
+	dir, err := ioutil.TempDir("", "zktrie-bench")
+	if err != nil {
+		panic(fmt.Sprintf("can't create temporary directory: %v", err))
+	}
+	diskdb, err := leveldb.New(dir, 256, 0, "", false)
+	if err != nil {
+		panic(fmt.Sprintf("can't create temporary database: %v", err))
+	}
+	config := &Config{Cache: 256, Preimages: true, Zktrie: true}
+	return dir, NewDatabaseWithConfig(diskdb, config)
+}
+
+const benchElemCountZk = 10000
+
+func BenchmarkZkTrieGet(b *testing.B) {
+	_, tmpdb := tempDBZK()
+	zkTrie, _ := NewZkTrie(common.Hash{}, NewZktrieDatabaseFromTriedb(tmpdb))
+	k := make([]byte, 32)
+	for i := 0; i < benchElemCountZk; i++ {
+		binary.LittleEndian.PutUint64(k, uint64(i))
+
+		err := zkTrie.TryUpdate(k, k)
+		if err != nil {
+			b.Error("TryUpdate error")
+			b.FailNow()
+		}
+	}
+
+	zkTrie.db.db.Commit(common.Hash{}, true, nil)
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		binary.LittleEndian.PutUint64(k, uint64(i))
+		_, err := zkTrie.TryGet(k)
+		if err != nil {
+			b.Error("TryGet fail")
+			b.FailNow()
+		}
+	}
+	b.StopTimer()
+
+	ldb := zkTrie.db.db.diskdb.(*leveldb.Database)
+	ldb.Close()
```

Comment:
maybe we can use `defer` here? To avoid quitting in other places. e.g., https://github.com/scroll-tech/go-ethereum/blob/df7ec192a04b647dc447a3e57dbc2f50d97d32a7/trie/zk_trie_test.go#L182